bridges/YGGTorrentBridge.php

```diff
@@ -111,7 +111,7 @@ public function collectData() {
 		foreach($results->find('tr') as $row) {
 			$count++;
 			if($count == 1) continue;
-			if($count == 12) break;
+			if($count == 22) break;
```

Comment:
Maybe name the constant or add a comment stating why this particular value is used

Reply:
I am to blame actually... :cry: 
I though there was a specific reason, but indeed this is just for the limit of items.